pkg/hub/submarineragent/controller.go

```diff
@@ -692,7 +693,37 @@ func (c *submarinerAgentController) createGNConfigMapIfNecessary(ctx context.Con
 		return errors.Wrapf(err, "error creating globalnet configmap on Broker")
 	}
 
-	return nil
+	// Ideally we should add option to have custom labels when creating configmap
+	return c.updateBackupLabelOnGnConfigMap(ctx, brokerNamespace)
```

Comment:
We don't actually need this. We can call `NewGlobalnetConfigMap` and create CM directly:

```
configMap, err := globalnet.NewGlobalnetConfigMap(brokerObj.Spec.GlobalnetEnabled,
		brokerObj.Spec.GlobalnetCIDRRange, brokerObj.Spec.DefaultGlobalnetClusterSize, brokerNamespace)
if err == nil {
    configMap.Labels[BackupLabel] = "submariner"
    err = client.Create(ctx, c.controllerClient)
}

return return errors.Wrapf(err, "error creating globalnet configmap on Broker")
```